### ISSUE TITLE
[cli] use Deno KV to persist flag values for better DX

### ISSUE THREAD

**Issue by miguelrk** [netzo/netzo#42]
**TLDR**: revent flag values like `--api-key` and `--project` could be persisted using Deno KV in unique database constructed from the file URL of the project root (e.g. using `Deno.openKv(import.meta.url)`?)

### Context

Refer to the [managing state](https://deno.com/blog/build-cross-platform-cli#managing-state) section of the [Build a Cross-Platform CLI with Deno in 5 minutes](https://deno.com/blog/build-cross-platform-cli) blog post.

![image](https://github.com/netzo/app/assets/18708856/f98a7bda-c154-40b1-8898-78db2e580fa3)

### Example

This example is also taken from the blogpost and simply used for illustrative purposes of how the code could look like.

```diff
- function main(inputArgs: string[]): void {
+ async function main(inputArgs: string[]): Promise<void> {

  const args = parseArguments(inputArgs);

  // If help flag enabled, print help.
  if (args.help) {
    printHelp();
    Deno.exit(0);
  }

  let name: string | null = args.name;
  let color: string | null = args.color;
  let save: boolean = args.save;

+  const kv = await Deno.openKv("/tmp/kv.db");
+  let askToSave = false;

+  if (!name) {
+    name = (await kv.get(["name"])).value as string;
+  }
+  if (!color) {
+    color = (await kv.get(["color"])).value as string;
+  }
+  if (save) {
+    await kv.set(["name"], name);
+    await kv.set(["color"], color);
+  }

  console.log(
    `%c${
      greetings[Math.floor(Math.random() * greetings.length) - 1]
    }, ${name}!`,
    `color: ${color}; font-weight: bold`,
  );
}
```
